dandi/tests/test_download.py

```diff
@@ -23,29 +23,32 @@ def test_download_multiple_files(monkeypatch, tmpdir):
     # to this test will test those retries.
     # While at it we will also test girder downloadFile to retry at least 3 times
     # in case of some errors, and that it sleeps between retries
-    orig_downloadFileAsIterator = GirderCli.downloadFileAsIterator
+    orig_sendRestRequest = GirderCli.sendRestRequest
```

Comment:
hm, for me none of the lines in this file got annotation for being covered or not from coveralls (just base red/green colors, not the vibrant ones as for annotating coverage)... is that what you see too @jwodder ? any idea why?

Reply:
If I run the tests from the command line, this file is reported as being 100% covered.  My best guess is that codecov might be disregarding coverage for test files.